tests/pgautofailover_utils.py

```diff
@@ -687,7 +687,7 @@ def stop(self):
 
             try:
                 pgid = os.getpgid(self.run_proc.pid)
-                os.killpg(pgid, signal.SIGTERM)
+                os.killpg(pgid, signal.SIGQUIT)
```

Comment:
Is this better than SIGTERM ? documentation says it is a signal to dump core

Reply:
We implement our own signal handlers in our code, see https://github.com/citusdata/pg_auto_failover/blob/master/src/bin/pg_autoctl/signals.c#L79 for the SIGQUIT handler. That's an _immediate shutdown_.